data/filedefs/filetypes.Arduino.conf

```diff
@@ -0,0 +1,56 @@
+# Arduino custom filetype
+#
+# By SukkoPera <software@sukkology.net>
+# 2016/12/14
```

Comment:
timestamp is recorded in Git, no need to add it here IMO, esp. as it'll get forgotten and then out of date in no time :)